csrc/ir/interface_nodes.h

```diff
@@ -272,6 +272,10 @@ struct CircularBufferOptions {
   int64_t stage = 0; // Size of the circular buffer (number of buffers)
   int64_t prefetch = 0; // Number of iterations ahead of the compute to
                         // prefetch, can only be < stage.
+  // The number of registers for load and compute warps respectively.
+  // Register sharing is disabled when both values are -1.
+  std::optional<std::pair<int64_t, int64_t>> warp_specialized_num_registers =
```

Comment:
Should we move this to `struct WarpSpecialized`?

Reply:
Done.